paper/src/main/java/vg/civcraft/mc/civmodcore/world/locations/chunkmeta/stat/PluginStatistic.java

```diff
@@ -0,0 +1,28 @@
+package vg.civcraft.mc.civmodcore.world.locations.chunkmeta.stat;
+
+public class PluginStatistic {
+	public final short pluginId;
+	public final String pluginName;
+	public long chunkLoadCount;
+	public long chunkLoadSumNanoSec;
+	public long chunkLoadMinTimeNanoSec;
+	public long chunkLoadMaxTimeNanoSec;
+
+	public boolean isInitialized;
+
+	public PluginStatistic(short pluginId, String pluginName) {
+		this.pluginId = pluginId;
+		this.pluginName = pluginName;
+	}
+
+	public PluginStatistic clone() {
```

Comment:
A private copy constructor might be a good implementation decision here.

PluginStatistic clone = new PluginStatistic(this);

Reply:
Good point.
Done.